### DISCUSSION TITLE
Metapacket for Debian12?

### DISCUSSION THREAD

**Discussion by VamLykTan** [brunoherbelin/vimix#128] [Q&A] [answered]
Hello everyone,

It's nice that there is now a successor to GLMixer, but as I can see from the information, it only seems to be available for Ubuntu, which I admit is a shame.
As a DJ/VJ, this is exactly the program I need for my streams.

Is a release planned for Debian 12 at some point, or would it make sense for an inexperienced CPP-User to continue looking for a program that he/she/it can use?

**Comment by brunoherbelin** [comment, marked as answer]
Hi ! Good if vimix can be what you need

vimix is available for all linux distribs through flatpack and as debian package (although the apt install will get you an old version)
https://github.com/brunoherbelin/vimix/wiki/Quick-Installation-Guide

**Comment by VamLykTan** [comment]
Unfortunately this didn't really help me under Debian 12 o.O

But under Ubuntu 22 it runs as it should and what I find to be an advantage over GLMixer is that images are not cropped.

**Reply by brunoherbelin** [reply]
> Unfortunately this didn't really help me under Debian 12 o.O

doesn't Debian 12 supports Flathub ? https://flathub.org/setup/Debian
